test/ex_force_test.exs

```diff
@@ -573,6 +573,193 @@ defmodule ExForceTest do
              {:error, :econnrefused}
   end
 
+  test "create_sobjects/3 - success", %{bypass: bypass, client: client} do
+    Bypass.expect_once(bypass, "POST", "/services/data/v40.0/composite/sobjects", fn conn ->
```

Comment:
these tests are a bit misleading - I believe the composite API is available post v42.0 (at least according to the historical [salesforce docs](https://developer.salesforce.com/docs/atlas.en-us.212.0.api_rest.meta/api_rest/resources_composite_sobjects_collections_delete.htm))